back/db/migrate/20220531123916_create_home_pages.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class CreateHomePages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :home_pages, id: :uuid do |t|
+      t.boolean :header_avatars_enabled, default: true, null: false
+
+      t.boolean :top_info_section_enabled, default: true, null: false
+      t.jsonb :top_info_section_multiloc, default: {}, null: false
+
+      t.boolean :bottom_info_section_enabled, default: true, null: false
+      t.jsonb :bottom_info_section_multiloc, default: {}, null: false
+
+      t.boolean :events_enabled, default: false, null: false
+
+      t.boolean :projects_enabled, default: true, null: false
+      t.jsonb :projects_header, default: {}, null: false
+
+      t.boolean :banner_enabled, default: true, null: false
+      t.string :banner_layout, default: 'full_width_banner_layout', null: false
+      t.string :banner_signed_in_header
+      t.string :banner_signed_in_text
```

Comment:
If it's the former `customized_button#text`, then yes, it should be multiloc.
And to avoid confusion in the future (working on this epic), it can be useful to describe the mapping of old and new fields. Or even better, preserve old names.

Reply:
I'll make this a multiloc, good catch Simon!

I think the naming can be improved, which is why I took the liberty to do so here: For example I think it makes sense to prefix all banner related settings with `banner`, because it brings clarity in what the setting belongs to.

I'll add the new and old tables to Notion in a second, good idea Alex!